src/govee_ble/parser.py

```diff
@@ -107,11 +110,19 @@ def _process_mfr_data(
         ):
             self.set_device_type("H5072/H5075")
             temp, humi = decode_temp_humid(data[1:4])
-            batt = int(data[4])
-            if temp >= MIN_TEMP and temp <= MAX_TEMP:
+            batt = int(data[4] & 0x7F)
+            err = data[4] & 0x80
```

Comment:
Since the error seems to resolve quickly, I think it would be better if we log the error at debug level and returned so they don't get an update if the device is in an error state.

Reply:
Yes, I am not sure about the logic here.
In my case the battery is still valid when the error bit it set.
I personally prefer the temperature and humidity to go to unavailable when the error bit is set. This matches the dashes in the display at the same time, and avoids preserving the same values for an extended period of time (flat-lining), up to one hour in my case. I also prefer unavailable for the automation I have for this temperature sensor.
I am not sure if this logic holds for `temp >= MIN_TEMP and temp <= MAX_TEMP`.